examples/actix-http-tracing/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "actix-http-tracing"
 version = "0.1.0"
 edition = "2018"
 publish = false
+license = "Apache-2.0"
```

Comment:
Instead of this, I usually set `publish = false` in `Cargo.toml` and then enable ignoring private crates in `deny.toml`. But maybe it's useful to explicitly license the examples in this case?

Reply:
I guess that depends on the policy of opentelemetry org. But we should add `publish = false` regardless